content/bn/docs/reference/glossary/contributor.md

```diff
@@ -0,0 +1,18 @@
+---
+title: অবদানকারী
+id: contributor
+date: 2018-04-12
+full_link: 
+short_description: >
+  কুবেরনেটিস প্রকল্প বা কমিউনিটিকে সহায়তার জন্য যিনি কোড, ডকুমেন্টেশন, বা তাদের সময় দান করেন।
+
+aka: 
+tags:
+- community
+---
+যে ব্যক্তি কুবেরনেটিস প্রকল্প বা কমিউনিটিকে সহায়তার জন্য কোড, ডকুমেন্টেশন, বা তাদের সময় দান করেন।
```

Comment:
![image](https://github.com/user-attachments/assets/4e42915f-44c6-465a-844a-69bd3014f6a4)

Kindly use the same localization for lines 7 and 13 as those are the same sentence.

You can use this localization on both lines. 
```
যে ব্যক্তি কুবারনেটিস প্রকল্প বা কমিউনিটিকে সহায়তার জন্য কোড, ডকুমেন্টেশন, বা তাদের সময় দান করেন।
```

Reply:
I think প্রজেক্ট will be more suitable instead of  প্রকল্প